workspaces/ui-v2/src/optic-components/diffs/render/ApproveAll.tsx

```diff
@@ -33,6 +33,9 @@ export default function ApproveAll(props: { disabled?: boolean }) {
   };
 
   const handleSave = () => {
+    if (shapeDiffs.loading) {
```

Comment:
This shouldn't be reachable (since we shouldn't reach this flow before having shapes to approve) - but if it does get reached - do we have a way of sending errors (or sentry alerts)?